### DISCUSSION TITLE
Add support for Folia servers

### DISCUSSION THREAD

**Discussion by KamiRori** [PotatoCraft-Studio/QuickShop-Reremake#382] [Ideas]
https://github.com/PaperMC/Folia

**Comment by YuanYuanOwO** [comment]
From Discord Channel:  
No currently, since its API is not complete and it's most likely need to rewrite the plugin (although quickshop is chunk-based management, its api is very different from bukkit api )

**Reply by GoodrichDev** [reply]
I have been going through a lot of rewrites of Paper based plugins into Folia and reviewing QuickShop, there isn't much that needs to be picked up. I think you can get it done pretty easily.

**Reply by YuanYuanOwO** [reply]
> I have been going through a lot of rewrites of Paper based plugins into Folia and reviewing QuickShop, there isn't much that needs to be picked up. I think you can get it done pretty easily.

There is no plan to add folia support at this time
#468
